bigtable/metricscaler/metricscaler.py

```diff
@@ -21,7 +21,24 @@
 from google.cloud import bigtable
 from google.cloud import monitoring
 
-import strategies
+
+_MIN_NODE_COUNT = 3
```

Comment:
ISTM that these constants are only used in one function. How do you feel about moving these into that function to avoid indirection?